tsfile/src/main/java/org/apache/iotdb/tsfile/tool/upgrade/UpgradeTool.java

```diff
@@ -83,12 +86,12 @@ public static void upgradeOneTsfile(String tsfileName, String updateFileName) th
   }
 
   public static void main(String[] args) throws IOException {
-    List<String> tsfileDirs = new ArrayList<>();
-    List<String> tsfileDirsUpdate = new ArrayList<>();
-    tsfileDirs.add("/Users/tianyu/incubator-iotdb/data/data/sequence/root.group_9");
-    tsfileDirsUpdate.add("/Users/tianyu/incubator-iotdb/data/data/sequence/root.group_10");
-    for (int i = 0; i < tsfileDirs.size(); i++) {
-      updateTsfiles(tsfileDirs.get(i), tsfileDirsUpdate.get(i));
+    List<String> oldVersionTsfileDirs = new ArrayList<>();
+    List<String> newVersionTsfileDirs = new ArrayList<>();
+    oldVersionTsfileDirs.add("/Users/tianyu/incubator-iotdb/data/data/sequence/root.group_10");
+    newVersionTsfileDirs.add("/Users/tianyu/incubator-iotdb/data/data/sequence/root.group_8");
+    for (int i = 0; i < oldVersionTsfileDirs.size(); i++) {
+      updateTsfiles(oldVersionTsfileDirs.get(i), newVersionTsfileDirs.get(i));
```

Comment:
How about making this a real tool.

Reply:
good idea,I will make it a sh script then.